core/tests/unit/ldml/test_context_normalization.cpp

```diff
@@ -109,12 +109,12 @@ void test_context_normalization_hefty() {
 }
 
 void test_context_normalization_invalid_unicode() {
```

Comment:
Rather than modifying this test, can we add a separate test for the initial trailing surrogate? 

Then of course we need to fixup this test anyway to have the trailing surrogate anywhere but 
at the start of the context! If you were truly pedantic, you'd include both unpaired leading and trailing surrogates mid-context.

(If we cannot get this test to pass as it stands, then we can leave it commented out, but I want to make sure we don't forget about cleanup of illegal chars in the context.)

Reply:
I only changed the test conditions not the input.